cadquery/occ_impl/shapes.py

```diff
@@ -1974,6 +2019,35 @@ def makeFromWires(
 
         return cls(face).fix()
 
+    def fillet2D(self, radius: float, vertices: Iterable[Vertex]) -> "Face":
+        """
+        Apply 2D fillet to a face
+        """
+
+        fillet_builder = BRepFilletAPI_MakeFillet2d(self.wrapped)
+
+        for v in vertices:
+            fillet_builder.AddFillet(v.wrapped, radius)
+
+        fillet_builder.Build()
+
+        return self.__class__(fillet_builder.Shape())
+
+    def chamfer2D(self, d: float, vertices: Iterable[Vertex]) -> "Face":
+        """
+        Apply 2D chamfer to a face
+        """
+
+        chamfer_builder = BRepFilletAPI_MakeFillet2d(self.wrapped)
+        edge_map = self._entitiesFrom("Vertex", "Edge")
+
+        for e1, e2 in edge_map.values():
+            chamfer_builder.AddChamfer(TopoDS.Edge_s(e1), TopoDS.Edge_s(e2), d, d)
+
+        chamfer_builder.Build()
+
+        return self.__class__(chamfer_builder.Shape()).fix()
```

Comment:
`chamfer2D` does not use the `vertices` argument?

It also errors with an open wire, like 
```python
w0 = cq.Workplane().hLine(1).vLine(1).wire().val().chamfer2D(0.2, [])
```
```
>       for e1, e2 in edge_map.values():
E       ValueError: not enough values to unpack (expected 2, got 1)

cadquery/occ_impl/shapes.py:2044: ValueError
```

Reply:
Should be resolved now. Thanks for spotting this!